flang/include/flang/Optimizer/Transforms/Passes.h

```diff
@@ -50,8 +49,7 @@ namespace fir {
 #define GEN_PASS_DECL_OPENACCDATAOPERANDCONVERSION
 #include "flang/Optimizer/Transforms/Passes.h.inc"
 
-std::unique_ptr<mlir::Pass> createAbstractResultOnFuncOptPass();
-std::unique_ptr<mlir::Pass> createAbstractResultOnGlobalOptPass();
+std::unique_ptr<mlir::Pass> createAbstractResultOptPass();
```

Comment:
This declaration can be auto-generated by TableGen by just removing the
`let constructor = ...` line in the Pass entry. This will also automatically create an overload that can be constructed with the option object.

Reply:
Thanks for this. I'll update the flang passes as I go along